### ISSUE TITLE
spikes[0] returns unexpected

### ISSUE THREAD

**Issue by sverhoeven** [matchms/matchms#54]
I tried to use the `spikes[i]`, but an unexpected result.

```python
import numpy as np
from matchms import Scores, Spectrum
from matchms.similarity import CosineGreedy

spectrum = Spectrum(mz=np.array([100, 150, 200.]), intensities=np.array([0.7, 0.2, 0.1]), metadata={'id': 'spectrum1'})

spectrum.peaks[0]
```

I expected

```python
(100.0, 0.7)
```

But I got

```python
[100. 150. 200.]
```

**Comment by jspaaks** [comment]
Just to provide some context, here is the part of the code that yields unexpected results:

https://github.com/matchms/matchms/blob/106b4cbc058b924aa9170f3f8cb32bcc49054011/matchms/Spikes.py#L30-L31

It was implemented like this as a way to unpack ``peaks`` and ``losses`` like
```python
mz, intensities = peaks
```
But maybe we should reevaluate that

**Comment by sverhoeven** [comment]
When I use `i` index I expect to get the spike (mz and intensity) at position `i` of the spikes array.

**Comment by jspaaks** [comment]
Not sure if it is conflicting, but we could have a ``__getitem__`` (probably not a ``__setitem__``) that returns ``mz[i]`` and ``intensities[i]`` as a tuple when you ask for ``spikes[i]``, and then combine that with something like https://stackoverflow.com/a/53009707 (basically, an ``__iter__`` that ``yield``s from an ad hoc list with ``[mz, intensities]``.

**Comment by florian-huber** [comment]
I wasn't sure at first, but I think I would also go for adding such a getter.
I had it now a few times that I wanted to get an 2D array of both mz and intensities, which according to Stefans suggestions would simply be accessible by ``peaks_array = spectrum.peaks[:]``.

That seems a bit nicer than ``peaks_array = np.asarray(spectrum.peaks[:])`` which I need to call right now.

**Comment by florian-huber** [comment]
Will be fixed with #291
